integration/spark2/src/test/scala/org/apache/spark/carbondata/restructure/AlterTableValidationTestCase.scala

```diff
@@ -523,7 +523,8 @@ class AlterTableValidationTestCase extends Spark2QueryTest with BeforeAndAfterAl
     }
   }
 
-  test("describe formatted for default sort_columns pre and post alter") {
+  // after changing default sort_scope to no_sort, all dimensions are not selected for sorting.
+  ignore("describe formatted for default sort_columns pre and post alter") {
```

Comment:
Instead of ignoring update the testcase by specifying the sort_columns while creating the table

Reply:
ok.

Reply:
ok. changed back to table properties

Reply:
Already one more just below tet case is doing same, so ignored this one.